src/View/Dish/index.html.twig

```diff
@@ -0,0 +1,39 @@
+{% extends 'layout-visitor.html.twig' %}
+
+
+
+{% block content %}
+    <section class="menupage">
+    <h1 class="menutitle">Notre Carte</h1>
+        <div class="menupricebox">
+            <p class="dishprices">Menu Petit Tonneau Entrée/Plat ou Plat/Dessert 22 €</p>
+            <p class="dishprices">Menu Grand Tonneau: Entrée/Plat/Dessert 28 €</p>
+        </div>
+    {% for category in categories %}
+        <h2 class="menusection">{{ category.name }}  : {{ category.price }}  € <img src="/assets/images/faitmaison.png" alt="fait maison logo" class="faitmaisonlogo"></h2>
+        {% for dishesWithCategory in dishesWithCategories %}
+            <ul class="menulist">
+            {% for dish in dishesWithCategory %}
+                {% if dish.name == category.name %}
+                    {% if (dish.sup != NULL) %}
```

Comment:
pas besoin de ce if non plus
fais ça, ça devrait fonctionner
` <li class="menutext">{{dish.dish_name}}{{ dish.sup is not null ? '<strong>(sup + ' ~ dish.sup ~ ' euros)</strong'}}</li>`

Reply:
ok